docs/guides/fedora.md

```
@@ -0,0 +1,267 @@
+![ROG Control Center](../assets/guides/shared/rog-control-center.png)
+
+![ROG Control Center fan curve](../assets/guides/shared/rog-control-center-fan-curve.png)
+
+> [!NOTE]
+> For complete functionality and driver support, it is recommended to use a Kernel version of 6.19 or greater.
+
+#### Install Nvidia Graphics Drivers
+
+> [!NOTE]
+> AMD dGPU laptop owners can skip this section.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)